### ISSUE TITLE
verb-generate-readme: get-pkg broken (used by various dependencies)

### ISSUE THREAD

**Issue by stefanwalther** [verbose/verb-generate-readme#23]
An issue in `get-pkg` actually makes this repo to break, see here:
https://github.com/jonschlinkert/get-pkg/issues/4

![image](https://user-images.githubusercontent.com/669728/40580499-d1890eb6-613f-11e8-8efb-4c9bc0bdb55f.png)

I have created a PR, but wasn't able to update the whole chain of requirements to get verb-generate-readme ...

Edit: I have found it, as soon as the get-pkg PR is merged, verb-repo-helpers need to be updated, then verb-generate-readme should again work as designed.

@jonschlinkert , @doowb , could you pls verify, thx a lot!

**Comment by stefanwalther** [comment]
**Update as of 2018-05-29**:

The following needs (at least) to be done to fix the issues:

- [x] Accept the PR of [jonschlinkert/get-pkg](https://github.com/jonschlinkert/get-pkg), bump to a new version
- [x]  Update [jonschlinkert/reflinks](https://github.com/jonschlinkert/reflinks) to point to the new version of get-pkg (PR is [here](https://github.com/jonschlinkert/reflinks/pull/4))
  - [x] Waiting for this to be merged and **bumped to a new version**
- [x]  Update [helper/helper-reflinks](https://github.com/helpers/helper-reflinks) to the new version  of [jonschlinkert/reflinks](https://github.com/jonschlinkert/reflinks) (v3.2.2)
   - [x] PR prepared [here and waiting to be merged](https://github.com/helpers/helper-reflinks/pull/5)
   - [x] waiting for reflinks to be bumped to a new version
- [x]  Update [helpers/helper-related](https://github.com/helpers/helper-related) to new version of [jonschlinkert/reflinks](https://github.com/jonschlinkert/reflinks)
  - [x] A [PR is prepared and waiting to be merged](https://github.com/helpers/helper-related/pull/10)
- [x]  Update [verb-repo-helpers](https://github.com/verbose/verb-repo-helpers) to point to the new version of [helper/helper-reflinks](https://github.com/jonschlinkert/reflinks)
  - [x] Waiting for helper-reflinks and helper-related to be bumped to a new version
  - [x] PR prepared [here](https://github.com/verbose/verb-repo-helpers/pull/2)
- [x] Update [jonschlinkert/gulp-reflinks](https://github.com/jonschlinkert/gulp-reflinks) to point to the newest version of 
- [x]  Update [verb-generate-readme](https://github.com/verbose/verb-generate-readme) to point to the new versions of verb-reflinks
  - [x] [jonschlinkert/gulp-reflinks](https://github.com/jonschlinkert/gulp-reflinks) version
- [x]  Update [verb-repo-helpers](https://github.com/verbose/verb-repo-helpers) to point to the new version of [jonschlinkert/get-pkg](https://github.com/jonschlinkert/get-pkg)
- [x]  Update verb-repo-helpers to new version of [helpers/helper-related](https://github.com/helpers/helper-related)
- [x] Update [verb/reflinks](https://github.com/verbose/verb-reflinks) to point to newest version of jonschlinkert/reflinks
- [x] Update gulp/reflinks
- [ ] Sugar: Update https://github.com/stefanwalther/docker-verb

I am happy to do all the PR for all the repos above as soon as the [jonschlinkert/get-pkg](https://github.com/jonschlinkert/get-pkg) PR is accepted and merged.

For those being desparate to get a fix right now, use https://github.com/stefanwalther/docker-verb, where I have provided an update by using forked versions of all the repos above (which I will update again as soon as the fixes are in place again).

**Comment by jonschlinkert** [comment]
@stefanwalther I can help get these resolved faster tomorrow. or/and we can add you to any of the mentioned projects and give you publish rights if you want. You are always awesome about following our coding and maintenance conventions, so I'd have no concerns about you doing it if you want.

**Comment by stefanwalther** [comment]
HI @jonschlinkert, as you want ... either add me or just follow the PRs ... whatever is more convenient to you ... there is a lot to update ;-) - unfortunately I am detecting more and more dependencies ...

**Comment by stefanwalther** [comment]
I see that you are working on it in parallel, so then I stop ... seems that we are working on the same takes ;-)

**Comment by jonschlinkert** [comment]
> I see that you are working on it in parallel, so then I stop

? I was merging your PRs. No worries if you don't want to do anymore. I just refactored 3 of the libs, and updated 5 of in the past hour or so. I think there are only two left. Should only take 10-15 min for me. My girls just got home though, so I'll do this later tonight or tomorrow.

**Comment by stefanwalther** [comment]
OK, sorry, then it was obviously too late ... continue tonight ... ;-)

**Comment by jonschlinkert** [comment]
Closing, I think the listed items are complete. Please feel free to re-open or create another issue if you find something we missed.

**Comment by stefanwalther** [comment]
Hmmm, still investigating, but with the recent changes I now get:

![image](https://user-images.githubusercontent.com/669728/40747886-f9988e5a-645e-11e8-872f-f840762495b5.png)

(I am running inside a docker-container, but that all worked nicely before; can't see any changes to data-store, can you remember something you have changed yesterday)

**Comment by stefanwalther** [comment]
More errors doing a normal install (npm install -g verbose/verb#dev verb-generate-readme && verb):

```
util.js:1193
  Object.setPrototypeOf(ctor.prototype, superCtor.prototype);
         ^

TypeError: Object.setPrototypeOf called on null or undefined
    at Function.setPrototypeOf (<anonymous>)
    at Object.inherits (util.js:1193:10)
    at Function.Template.extend (/usr/local/lib/node_modules/verb/node_modules/template/index.js:795:8)
    at Object.<anonymous> (/usr/local/lib/node_modules/verb/index.js:27:10)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Module.require (internal/modules/cjs/loader.js:650:17)

```

**Comment by stefanwalther** [comment]
More errors, now installing node.js globally on my mac (using `npm install -g verbose/verb#dev verb-generate-readme`):

Running `verb`:
```
util.js:951
  Object.setPrototypeOf(ctor.prototype, superCtor.prototype);
         ^

TypeError: Object.setPrototypeOf called on null or undefined
    at Function.setPrototypeOf (<anonymous>)
    at Object.inherits (util.js:951:10)
    at Function.Template.extend (/usr/local/lib/node_modules/verb/node_modules/template/index.js:795:8)
    at Object.<anonymous> (/usr/local/lib/node_modules/verb/index.js:27:10)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)

```
Happens on both: node.js v10.3 + node.js v9.3

Using the following verb-config in package.json:

```
"verb": {
    "run": true,
    "toc": true,
    "layout": "empty",
    "tasks": [
      "readme"
    ],
    "plugins": [
      "gulp-format-md"
    ],
    "related": {
      "list": [ ]
    },
    "lint": {
      "reflinks": true
    },
    "reflinks": [
    ]
  }
```

**Comment by jonschlinkert** [comment]
> (I am running inside a docker-container, but that all worked nicely before; can't see any changes to data-store, can you remember something you have changed yesterday)

Yes, we bumped major versions on a bunch of libraries ;). 

If `data-store` is trying to create a directory named `/` then I'm guessing the `store.path` is `/`, which is wrong. 

Do you have any custom code that is using data-store? If not, then we'll need more information to track down where it's being used, so I can fix how the path is being passed. 

@doowb?

**Comment by stefanwalther** [comment]
No custom code ... will try to debug resp provide some more information!
